File: php/root_command/Variable.php
<?php

/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Scripting/PHPClass.php to edit this template
 */

/**
 * Description of Variable
 *
 * @author Administrator
 */
class Variable {

    //put your code here
    public $interpreter;

    public function __construct($interpreter) {
        $this->interpreter = $interpreter;
    }

    public function load_command($command_name, $name, $global = "") {
        if ($globale == "global") {
            $this->interpreter->global_variable[command_name] =
                    $this->interpreter->global_class[name]($this->interpreter->global_variable,
                            $this->interpreter->global_class);
        } else {
            $this->interpreter->global_variable[command_name] =
                    $this->interpreter->global_class[name]();
        }
    }
}

File: LineUp/python/LineUp/root_command/variable.py
class Variable:
    def __init__(self, interpreter):
        self.interpreter = interpreter

    def load_command(self, command_name, name, globale=""):
        if globale == "global":
            self.interpreter.global_variable[command_name] = \
                self.interpreter.global_class[name](self.interpreter.global_variable,
                                                    self.interpreter.global_class)
        else:
            self.interpreter.global_variable[command_name] = \
                self.interpreter.global_class[name]()
